pymc/distributions/continuous.py

```diff
@@ -2864,11 +2872,11 @@ class ExGaussian(Continuous):
 
     Parameters
     ----------
-    mu: float
+    mu : tensor_like of float, default 0
         Mean of the normal distribution.
-    sigma: float
+    sigma : tensor_like of float, optional
         Standard deviation of the normal distribution (sigma > 0).
-    nu: float
+    nu : tensor_like of float, optional
```

Comment:
These don't look optional either even if being set to None in the call signature